cocos/base/CCEventMouse.h

```diff
@@ -189,6 +196,8 @@ class CC_DLL EventMouse : public Event
     friend class EventListenerMouse;
 };
 
+std::ostream& operator<<(std::ostream& out, const EventMouse::MouseButton& button);
```

Comment:
I'm sure there's no good way to separate it out, but yeah it might be best left for game developers to add in their own code base instead of in the engine source.

Or maybe there's a good way to put all ostream << operator overloads into a separate CCStreamOperators.h might be better (though I can imagine that would require a lot of includes as part of the header one for each type that gets an overload ...??